src/Encamina.Enmarcha.SemanticKernel.Abstractions/IChatHistoryProvider.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.SemanticKernel.ChatCompletion;
+
+namespace Encamina.Enmarcha.SemanticKernel.Abstractions;
+
+/// <summary>
+/// Represents a collection of functions to work chat history.
+/// </summary>
+public interface IChatHistoryProvider
+{
+    /// <summary>
+    /// Loads chat history messages.
+    /// </summary>
+    /// <remarks>
+    /// The maximum number of messages to load is configured in <c>ChatHistoryProviderOptions.HistoryMaxMessages</c>.
```

Comment:
Maybe we shouldn't have this `remarks` here. Someone might create their implementation of this interface, and this could cease to be true :S